README.md

```diff
@@ -627,6 +630,37 @@ ServiceNow.getAllAttachmentsMetaData('a83820b58f723300e7e16c7827bdeed2', (res) =
 ___
 
 
+### 12. ServiceNow.getChangeTasks
+
+This is used to get change tasks from a change request in ServiceNow. (#12-getchangetasks)
+
+| Parameters        | Description                                                |
+|-------------------|------------------------------------------------------------|
+| type              | type of table - change_request                             |
```

Comment:
Type of table in the parameter is ``change_task``

Reply:
wrong copy past :) fixed